app/controllers/results_controller.rb

```diff
@@ -6,16 +6,26 @@ def create
     redirect_to result_path(assessment_code:)
   end
 
+  def early_result
+    @previous_step = params[:step].to_sym
+    session_data["api_response"] = CfeService.call(session_data, Steps::Helper.completed_steps_for(session_data, @previous_step))
+    set_early_result_type
+    redirect_to result_path(assessment_code:)
+  end
+
   def show
+    @early_assessment = session_data.dig("early_result", "type")
```

Comment:
can we rename this to early_result_type given that it is returning `gross`, and i assume in the future it will also be bale to return `disposable`.

I just think using early_assessment and early_result is confusing

Reply:
yes that name is much clearer!

Reply:
now changed :)